### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by Adafede** [luigiquiros/inventa#19] [Ideas]
Hi @luigiquiros!

Hereafter are some additional things I could think of:

- [x] Why are you transforming the LC to discrete values? (0,0.5,1), why not making them vary between 0 and 1 in a continuous way?
- [x] Why is the minimal specificity of FC expressed as % and not 0-1 based? (90 and not 0.9, looks it is the only one)
- [ ] The FC has quite a lot of comparison compared to other components and consists in reality of two scores, splitting might help?
- [ ] Passing arguments from the command line or through an external file outside of the notebook could be great!

Best,

**Comment by luigiquiros** [comment]
- [x] Why are you transforming the LC to discrete values? (0,0.5,1), why not making them vary between 0 and 1 in a continuous  way?

R/ done! a good idea to avoid a meta score. i just use a simple arithmetic equation to get a continous value! 
Is simpler than before, just a value of maximum reported compounds is needed.

**Comment by luigiquiros** [comment]
- [ ] Why is the minimal specificity of FC expressed as % and not 0-1 based? (90 and not 0.9, looks it is the only one)

R/ corrected!  now everything is 0-1!
